pkg/buses/stub/bus.go

```diff
@@ -52,7 +52,7 @@ func NewStubBusDispatcher(busRef buses.BusReference, opts *buses.BusOpts) *StubB
 		},
 		ReceiveMessageFunc: func(channelRef buses.ChannelReference, message *buses.Message) error {
 			bus.logger.Infof("Recieved message for %q channel", channelRef.String())
-			bus.channel(channelRef).receiveMessage(message)
+			bus.channel(channelRef).receiveMessage(bus, message)
```

Comment:
I'd be more inclined to capture the lodger when creating the stubChannel rather then as a function param.

Sorry, this is a moving target. If you'd rather I'd be ok with taking this as is and following up later. (Of course someone with approval rights will need to agree)

Reply:
Not a problem - I can update with adding the logger the the stubChannel struct.